example/spec/helpers/testUtils.js

```diff
@@ -137,7 +137,7 @@ const loadConfig = async (type = 'app') => {
   await loadEnvFile('./.env');
   verifyRequiredEnvironmentVariables();
 
-  const configFromFile = process.env.KES_DEPLOYMENT.toString() === 'true' ?
```

Comment:
If the `KES_DEPLOYMENT` environment variable is not set, this will cause tests to fail. Removing `toString()` achieves the desired behavior.